vlsi/Makefile

```diff
@@ -28,11 +28,16 @@ ifeq ($(tech_name),asap7)
 else
 	MACROCOMPILER_MODE ?= -l $(SMEMS_CACHE) -hir $(SMEMS_HAMMER)
 endif
-OBJ_DIR            ?= $(vlsi_dir)/build
-ENV_YML            ?= $(vlsi_dir)/env.yml
-INPUT_CONFS        ?= example.yml
+ENV_YML            ?= $(vlsi_dir)/bwrc-env.yml
```

Comment:
This should be left as `env.yml` for Chipyard. Ditto not including tech-plugin's `bwrc.yml` below.

Reply:
Probably can just go into VLSI documentation under `Tutorial` or `Advanced-Usage` to teach people how to populate these ymls.

Reply:
My bad, I copy-pasted all the lines but forgot that Chipyard is a bit different.